enterprise/ha/src/main/java/org/neo4j/kernel/ha/lock/SlaveLocksClient.java

```diff
@@ -256,36 +277,32 @@ private boolean getReadLockOnMaster( Locks.ResourceType resourceType, long resou
             makeSureTxHasBeenInitialized();
 
             RequestContext requestContext = newRequestContextFor( this );
-            try ( Response<LockResult> response = master.acquireSharedLock( requestContext, resourceType, resourceId ) )
+            try ( Response<LockResult> response = master.acquireSharedLock( requestContext, resourceType, resourceIds ) )
             {
-                return receiveLockResponse( response );
+                receiveLockResponse( response );
             }
             catch ( ComException e )
             {
                 throw new DistributedLockFailureException( "Cannot get shared lock on master", master, e );
```

Comment:
Comment is about message in exception that is wrong now, sorry that I was not clear enought